### DISCUSSION TITLE
R star MCMC convergence metric

### DISCUSSION THREAD

**Discussion by ben18785** [pints-team/pints#1657] [Ideas/Science]
This is a slightly self-involved one but I do actually think it'd be useful. I would like to add my R* convergence metric that I describe in [this](https://arxiv.org/abs/2003.07900) paper, as I've found it works pretty well across all problems I've thrown at it; it also offers additional information about convergence relative to the standard Rhat stats...anyway, enough sell!

One thing it requires is the ability to train a gradient-boosted regression tree ML model. It looks like Scikit-learn has this capability. How would people feel about having this as an optional dependency?

**Comment by MichaelClerx** [comment]
What's a gradient-boosted regression tree ML model?
